float32 left_front_coxa
float32 left_front_femur
float32 left_front_tibia

float32 right_front_coxa
float32 right_front_femur
float32 right_front_tibia

float32 left_middle_coxa
float32 left_middle_femur
float32 left_middle_tibia

float32 right_middle_coxa
float32 right_middle_femur
float32 right_middle_tibia

float32 left_rear_coxa
float32 left_rear_femur
float32 left_rear_tibia

float32 right_rear_coxa
float32 right_rear_femur
float32 right_rear_tibia
